pkg/command/command.go

```diff
@@ -79,7 +76,7 @@ func Run() error {
 				backoff.NewExponentialBackOff(),
 				func(_ error, dur time.Duration) {
 					logger.Warn("Database ping failed",
-						"retry", dur,
+						"retry", dur, "err", err,
```

Comment:
This would be nil as the error var in this func is underscore.

You are reformatting the code again 😂

Reply:
In my defence it was late and I was annoyed. 😢  Updated the PR now. I guess I have to disable editor formatting for this repo in my nvim config to avoid further mishaps :)